config/cops.yml

```diff
@@ -1,24 +1,26 @@
 Metrics/AbcSize:
-  violation_points: 100_000
+  base_points: 1_000_000
+  violation_points: 70_000
 Metrics/BlockNesting:
-  base_points: 30_000
+  base_points: 100_000
 Metrics/ClassLength:
-  base_points: 400_000
-Metrics/CyclomaticComplexity:
-  base_points: 75_000
-  violation_points: 10_000
+  base_points: 5_000_000
+  violation_points: 35_000
+Metrics/CyclomaticComplexity: # This check is per method
+  base_points: 1_000_000
+  violation_points: 70_000
 Metrics/LineLength:
-  base_points: 20_000
-  violation_points: 5_000
-Metrics/MethodLength:
-  base_points: 50_000
+  base_points: 100_000
```

Comment:
Is `LineLength` for a single LOC being too long (wide)? If so, this seems too high.